packages/p2p/src/stores/order-store.js

```diff
@@ -139,14 +140,15 @@ export default class OrderStore {
                         response?.error.code === api_error_codes.INVALID_VERIFICATION_TOKEN ||
                         response?.error.code === api_error_codes.EXCESSIVE_VERIFICATION_REQUESTS
                     ) {
+                        clearTimeout(wait);
                         // TODO: remove this once refactoring of EmailLinkVerifiedModal is done
                         this.root_store.general_store.hideModal();
                         this.setVerificationLinkErrorMessage(response.error.message);
-                        this.root_store.general_store.showModal({
-                            key: 'InvalidVerificationLinkModal',
-                            props: { error_message: response.error.message, order_id: id },
-                        });
+                        const wait = setTimeout(() => this.setIsInvalidVerificationLinkModalOpen(true), 230);
```

Comment:
wait how come you changed the showModal changes here for invalid-verification-link-modal?